src/components/Pages/EventList/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useRouteData, Head } from 'react-static';
+import moment from 'moment-timezone';
+import { useIntl } from 'react-intl';
+
+import PageHeader from 'components/PageHeader';
+import EventListPagination from 'components/Pages/EventList/EventListPagination';
+import UserFeedback from 'components/UserFeedback';
+import { events as i18n } from 'js/i18n/definitions';
+
+const filterOnDate = event => {
+  // the api returns events greater than or equal to the time when the site is built
+  // this filters out the ones between the build time and now
+  const dateNow = moment().format('YYYY-MM-DD')
+  return moment(event.date).isSameOrAfter(dateNow);
+}
```

Comment:
Should I limit this to be the current time in Austin? Or filter it based on wherever the user is? I am thinking Austin, but wanna hear feedback

Reply:
This is a good question. I think content would have a good point of view. I'd lean keeping it local time with CST added. for clarification. Like, if i'm in another time zone, i'm already thinking about that and to see CST i know for sure. I'm often more confused when it's updated to my current location while traveling. But, maybe good for this iteration and a thought down the road...

Reply:
That's a good content question. It might be confusing to adjust the eventList based on the user's location because we're not adjusting the individual event datetimes based on the user's location. That would be a bad user experience. Someone from New York looks at the site, or is planning a trip schedule, whatever. "Oh there's an event at 5:00 on the 26th." Intuitively they'd expect that to be 5:00 Texas time. That's the upside of making a site that's limited to only one city.

Reply:
shameless https://github.com/cityofaustin/techstack/issues/3897 plug